plugins/automation/examples/mark-as-danger.yaml

```diff
@@ -0,0 +1,10 @@
+name: "Mark Trivy vulnerabilities as danger"
+description: "Trivy vulnerabilities in namespaces that start with acme-app- should be marked as danger"
+context: ""
+reportType: "trivy"
+cluster: ""
+repository: ""
+action: |
+  if (ActionItem.ResourceNamespace.startsWith("acme-app-")) {
+     ActionItem.Severity = 0.667;
```

Comment:
Do we provide constants for danger? It might be nice

Reply:
Good catch. I'll add it in Insights and make a new PR to fix it here.